NetKAN/KerbalAircraftExpansion.netkan

```diff
@@ -1,15 +1,20 @@
 {
-	"spec_version" : "v1.4",
+	"spec_version" : "v1.12",
 	"identifier" : "KerbalAircraftExpansion",
 	"$kref" : "#/ckan/kerbalstuff/391",
 	"license" : "GPL-3.0",
 	"x_netkan_license_ok": true,
+	"x_netkan_epoch": "1",
 	"install" : [
 		{
 			"file": "KAX",
 			"install_to": "GameData",
-			"filter" : ["KAX 2.4.2 dds textures.7z", "Example Craft"],
-			"comment": "Example crafts are filtered out becouse its impossible to install ships to subdirectory."
+			"filter" : "Example Craft",
+			"comment": "Example crafts are installed to SPH subdirectory instead."
+		},
+		{
+			"find": "Example Craft",
+			"install_to": "Ships/SPH"
```

Comment:
This will not work as we cannot create `Ships/SPH/Example Craft/`. .craft files must be installed directly to `Ships/SPH/`. Thus we filter the Example Craft.